src/basic/log.c

```diff
@@ -1457,6 +1505,9 @@ void log_received_signal(int level, const struct signalfd_siginfo *si) {
                          "Received SIG%s from PID %"PRIu32" (%s).",
                          signal_to_string(si->ssi_signo),
                          si->ssi_pid, strna(p));
+
+                if(si->ssi_signo == SIGINT)
+                        log_task_chain(si->ssi_pid, "reboot");
```

Comment:
Also here. This is highly specific for pid 1 and doesn't belong here.

Reply:
Will the switch/case in `manager_dispatch_signal_fd()` be a better place for this then?